src/node/txreconciliation.cpp

```diff
@@ -17,6 +19,41 @@ namespace {
 const std::string RECON_STATIC_SALT = "Tx Relay Salting";
 const HashWriter RECON_SALT_HASHER = TaggedHash(RECON_STATIC_SALT);
 
+/**
+ * Announce transactions via full wtxid to a limited number of inbound and outbound peers.
+ * Justification for these values are provided here:
+ * https://github.com/naumenkogs/txrelaysim/issues/7#issuecomment-902165806 */
+constexpr double INBOUND_FANOUT_DESTINATIONS_FRACTION = 0.1;
+constexpr double OUTBOUND_FANOUT_DESTINATIONS = 1;
+/**
+ * If there's a chance a transaction is not streamlined along the first couple hops, it would take
+ *  very long to relay.
+ */
+static_assert(OUTBOUND_FANOUT_DESTINATIONS >= 1);
```

Comment:
>Every transaction should be assigned to some peer." a hard requirement?

I meant to relay in *some way* (either flooded or reconciled), so that the transaction doesn't get stuck. That's it.
That resolves your question right? Fixing comment for now.